src/test/codewhisperer/util/runtimeLanguageContext.test.ts

```diff
@@ -281,8 +281,14 @@ describe('runtimeLanguageContext', function () {
             ['scala', 'scala'],
             ['shell', 'shell'],
             ['sql', 'sql'],
+            ['sql', 'sql'],
+            ['tf', 'tf'],
+            ['hcl', 'tf'],
+            ['json', 'json'],
+            ['yaml', 'yaml'],
+            ['yml', 'yaml'],
             ['plaintext', 'plaintext'],
-            ['arbitrary string', 'arbitrary string'],
+            // ['arbitrary string', 'arbitrary string'],
```

Comment:
Can this be removed?

Reply:
I need @Will-ShaoHua input on this, Can this be removed from the test case as we are testing for all the other use cases.

Reply:
any reason why we need to comment this out?

Reply:
if it's not causing compile error or any error I don't get the reason why we have this change here

Reply:
Can I know what it does ?

Reply:
it's just a test case. 
```
public mapToRuntimeLanguage<
        T extends codewhispererClient.ListRecommendationsRequest | codewhispererClient.GenerateRecommendationsRequest
    >(request: T): T
```

type of `T.fileContext.programmingLanguage.languageName` is `string`, we can't gurantee it's something we must know in advance

Reply:
of course you can remove it IF we update and make our logic more robust by passing type of `CodewhispererLanguage` or something which will show error at compile time.

Reply:
although this method `mapToRuntimeLanguage` is another method we should refactor and remove, it's not covered by this PR so let's keep it as is now